index.html

```diff
@@ -0,0 +1,324 @@
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </main>
+
+    <script
+      src="https://kit.fontawesome.com/e966a3bd22.js"
+      crossorigin="anonymous"
+    ></script>
+    <div class="form-section">
+      <form action="https://formspree.io/f/mdoybpkz" method="post">
+          <ul class="form-list">
+              <li>
+                  <label for="name"></label>
+                  <input type="text" id="name" name="user_name" class="name-section" maxlength="30"
+                      placeholder="Name" >
+
+              </li>
+              <li>
+                  <label for="mail"></label>
+                  <input id="mail" name="user_email" class="email-section" placeholder="Email">
+              </li>
+              <li>
+                  <label for="msg"></label>
+                  <textarea id="msg" name="user_message" class="message-section"
+                      placeholder="Write Your Message here" required></textarea>
+              </li>
+              <li>
+                  <button type="submit" class="getintouch">Get In Touch</button>
+              </li>
```

Comment:
Kindly make sure the form button is positioned in the same way in the  [Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

![image](https://user-images.githubusercontent.com/43564324/157616667-206e396d-2c88-477b-b8d6-cb9439a625cc.png)